gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notifications-list.component.ts

```diff
@@ -113,4 +126,40 @@ export class NotificationsListComponent implements OnInit {
       )
       .subscribe(() => this.ngOnInit());
   }
+
+  addNotification() {
+    this.matDialog
+      .open<NotificationsAddNotificationDialogComponent, NotificationsAddNotificationDialogData, NotificationsAddNotificationDialogResult>(
+        NotificationsAddNotificationDialogComponent,
+        {
+          width: GIO_DIALOG_WIDTH.MEDIUM,
+          data: {
+            notifier: this.notifiersGroup,
+          },
+          role: 'dialog',
+          id: 'addNotificationDialog',
+        },
+      )
+      .afterClosed()
+      .pipe(
+        filter((result) => !!result),
+        switchMap((newNotification) =>
+          this.notificationSettingsService.create(this.ajsStateParams.apiId, {
+            ...this.newNotificationTemplate,
+            ...newNotification,
+            config_type: 'GENERIC',
+            hooks: [],
```

Comment:
What do you think about moving it into the dialog. so that it is responsible for on thing. Create NewNotificationSettings

Reply:
added